docs/integrations/cloudflare-pages.md

```diff
@@ -68,11 +68,11 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 10
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
 
-      - uses: actions/setup-node@v2
+      - uses: actions/setup-node@v4
         with:
-          node-version: 16
+          node-version: 20
```

Comment:
once we merge this PR, may I please request you to send one here for the Cloudflare example too? https://github.com/featurevisor/featurevisor-example-cloudflare

this way both the docs and implementation can stay in sync for others.

Reply:
Of course!

Reply:
actually thinking about it more, it would be wise to get the PR in example repo merged first. 

that way we confirm everything works as expected before publishing the docs.

Reply:
Done